forge-gui/res/cardsfolder/upcoming/the_tale_of_tamiyo.txt

```diff
@@ -0,0 +1,14 @@
+Name:The Tale of Tamiyo
+ManaCost:2 U
+Types:Legendary Enchantment Saga
+K:Chapter:4:DBRepeat,DBRepeat,DBRepeat,DBChangeZone
+SVar:DBRepeat:DB$ Repeat | RepeatSubAbility$ DBCleanAndGrind | MaxRepeat$ MaxRepeats | RepeatCheckSVar$ MilledSharesType | RepeatSVarCompare$ GE2 | SubAbility$ DBCleanup | SpellDescription$ Mill two cards. If two cards that share a card type were milled this way, draw a card and repeat this process.
+SVar:DBCleanAndGrind:DB$ Cleanup | ClearRemembered$ True | SubAbility$ DBGrind
+SVar:DBGrind:DB$ Mill | NumCards$ 2 | NoReveal$ True | RememberMilled$ True | SubAbility$ DBDraw
+SVar:DBDraw:DB$ Draw | ConditionCheckSVar$ MilledSharesType | ConditionSVarCompare$ GE2
+SVar:MaxRepeats:Count$InYourLibrary
+SVar:MilledSharesType:Remembered$Valid Card.sharesCardTypeWithOther Remembered

```

Comment:
I don't think the "Other" part exists for this one
I'll have to think if it can be written another way

Reply:
OK, that must be while it hanged in testing :)